submissions/2021-11-08-QMDataset-pubchem-set1-single-points/README.md

```diff
@@ -0,0 +1,50 @@
+## Description
+
+This is a single point energy calculation of PubChem set 1 (1-2500) molecules. Detailed description on how the data is generated can be found at https://github.com/openmm/qmdataset/tree/main/pubchem.
+
+## General Information
+
+ - Date: 2021.11.08
+ - Class: Basic dataset 
+ - Purpose: Energy calculation
+ - Collection: BasicDataset
+ - Name: SPICE PubChem Set 1 Single Points Dataset
+ - Number of unique molecules:        2372
+ - Number of filtered molecules:      0
+ - Number of conformers:              118606
+ - Number of conformers min mean max: 6  50.00 100
```

Comment:
??? If it has more than 50 conformers for any molecule, something is very strange!

Reply:
@dotsdl @jthorton any thoughts?

Reply:
I ran the following test just to be sure.

```python
import h5py
f = h5py.File('pubchem-1-2500.hdf5')
for id in f:
  conf = f[id]['conformations']
  if conf.shape[0] > 50:
    print(id, conf.shape)
```

It doesn't report anything.

Reply:
@peastman Yeah, I checked the same. During dataset preparation qcsubmit does a deduplication step (based on inchi keys IIRC), and the output shows a drop from 2373 to 2372 molecules. I checked the inchi keys of the created molecules, and for these two molecules with smiles `'[O-][P-](=O)[O-]', '[O-]P([O-])[O-]'`, inchi key is same, `'AQSJGOWTSHOLKH-UHFFFAOYSA-N'`. Somewhat related to this issue https://github.com/openforcefield/openff-qcsubmit/issues/60

Reply:
Is that going to cause problems?

Reply:
afaik it shouldn't matter since QM doesn't care about the 2D representation, and that one molecule record seems to have 100 conformers post dataset preparation

Reply:
Thanks for looking into this @pavankum I think you are right about the deduplication. Do we want to remove some of these conformers for this molecule so it only has 50 like the others.

Reply:
It's not a problem if there are more conformers.  It was just surprising.  And normally my filtering by ECFP4 fingerprint would have caught identical molecules, but this case is different because they really are the same as far as QM is concerned (same elements, same total charge), but the fingerprints are totally different.

Reply:
@peastman Thank you for going through the PR and noticing this detail, marking this as resolved now.